### PR TITLE
DATAUP-512: use batch_id instead of parent_job_id

### PR DESCRIPTION
**Pull request by ialarmedalien**
# Description of PR purpose/changes

* Switch front and backend over to using "batch id" instead of "parent_job_id" to refer to the batch job parent

# Jira Ticket / Issue #
e.g. https://kbase-jira.atlassian.net/browse/DATAUP-512
- [x] Added the Jira Ticket to the title of the PR (e.g. `DATAUP-69 Adds a PR template`)

# Testing Instructions
* Details for how to test the PR:
- [x] Tests pass locally and in GitHub Actions
- [x] Changes available by spinning up a local narrative running against CI, and then kicking off a batch job. Prior to the update, you would get a frontend error about batch jobs requiring a parent job ID and child IDs; this should disappear after this update.

# Dev Checklist:

- [x] My code follows the guidelines at https://sites.google.com/lbl.gov/trussresources/home?authuser=0
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] (JavaScript) I have run Prettier and ESLint on changed code manually or with a git precommit hook
- [x] (Python) I have run Black and Flake8 on changed Python code manually or with a git precommit hook

### CODE REVIEW

File: kbase-extension/static/kbase/js/widgets/narrative_core/jobCommChannel.js [hunk 1]
```diff
@@ -497,7 +496,9 @@ define([
                 ],
             });
 
-            $modalBody.find('button#kb-job-err-report').click(() => {});
+            $modalBody.find('button#kb-job-err-report').click(() => {
+                // no action
```
Comment (ialarmedalien) [inline kbase-extension/static/kbase/js/widgets/narrative_core/jobCommChannel.js:500]: keep eslint quiet

File: test/unit/spec/narrative_core/jobCommChannel-spec.js [hunk 2]
```diff
@@ -132,20 +132,18 @@ define(['jobCommChannel', 'base/js/namespace', 'common/runtime', 'testUtil'], (
             },
             {
                 channel: 'request-job-status',
-                message: { jobId: 'someJob', parentJobId: 'someParent' },
```
Comment (ialarmedalien) [inline test/unit/spec/narrative_core/jobCommChannel-spec.js:135]: I removed all the references to `parentJobId` in these tests as the backend no longer takes `parent_job_id` as a parameter for these queries.

File: test/unit/spec/nbextensions/bulkImportCell/bulkImportCell-spec.js [hunk 3]
```diff
@@ -6,8 +6,7 @@ define([
     'testUtil',
     '/test/data/testAppObj',
     'common/ui',
-    'json!/test/data/NarrativeTest.test_input_params.spec.json',
```
Comment (ialarmedalien) [inline test/unit/spec/nbextensions/bulkImportCell/bulkImportCell-spec.js:9]: `TestAppSpec` not used in this file.

**Review by briehl — APPROVED** [review]
Just looks like some updates to some key names, right? LGTM.

**Pull request merged**